File: pset1/mario_less/mario.c
#include<stdio.h>
#include<cs50.h>

int main(void)
{
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    for (int i = 1; i <= height; i++)
    {
        int number = height - i;

        for (int j = 0; j < number; j++)
        {
            printf(" ");
        }

        for (int k = 0; k < i; k++)
        {
            printf("#");
        }
        printf("\n");
    }

}

File: pset6/sentimental-mario-more/mario.py
# TODO

from cs50 import get_int

# get height
while True:
    number = get_int("Height: ")
    if number > 0 and number < 9:
        break

# initialize row
row = 1

# print mario
for i in range(number):
    for i in range(number - row):
        print(" ", end="")
    for i in range(row):
        print("#", end="")
    print("  ", end="")
    for i in range(row):
        print("#", end="")
    row += 1
    print()